### DISCUSSION TITLE
Cannot downgrade

### DISCUSSION THREAD

**Discussion by Barefoot-Dev** [vercel/community#5200] [Account Issue]
### Question

I need to downgrade from Pro to Hobby, but it says my team will be deleted.

Does that mean all my projects will be deleted too? If so, why? Is it possible to downgrade without deleting everything??

(I tried to upload this as a case, but it was bugged and kept saying invalid project, even though there was no project dropdown, so here I am...)

**Comment by amyegan** [comment]
Thanks for reaching out. You can [transfer your projects](https://vercel.com/docs/projects/overview#transferring-a-project) from Pro to Hobby before deleting the team. You can also [transfer your domains](https://vercel.com/docs/projects/domains/transfer-your-domain#) if you have any custom ones set up. Downgrading without deleting the team is not possible right now.

I'll let the team know you had trouble with the support form. If you wouldn't mind letting me know which Problem Area you chose, that will help direct investigation into the cause. 🙏

**Reply by bewallyt** [reply]
@amyegan So I'm an idiot and I deleted the team to downgrade from pro to hobby assuming the projects would still exist? i.e., they were originally on my hobby account.

Is it possible to get access to these projects or access back to the team itself again? For context, it was pretty much a PRO team of just my account.

**Reply by bewallyt** [reply]
@amyegan FWIW i think the team still exists because I can't create the identical team name again?
